include/aws/http/websocket.h

```diff
@@ -153,7 +153,7 @@ struct aws_websocket_client_connection_options {
      * Optional
      * Configuration options related to http proxy usage.
      */
-    struct aws_http_proxy_options *proxy_options;
+    const struct aws_http_proxy_options *proxy_options;
```

Comment:
`tls_options` above this line should also be const

Reply:
added tls_options to constified members